### PR TITLE
Jbws 4091 travis ci jobs

### PR DESCRIPTION
**Pull request by rsearls**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+matrix:
+  exclude:
+    - jdk: oraclejdk9
+      env: SERVER_VERSION=wildfly1010
+    - jdk: oraclejdk9
+      env: SERVER_VERSION=wildfly1000
+    - jdk: oraclejdk9
+      env: SERVER_VERSION=wildfly1100
+    - jdk: oraclejdk9
+      env: SERVER_VERSION=wildfly1110
```
Comment (asoldano) [inline .travis.yml]: For WildFly 11.0.0.Final we should not be excluding the jdk9 run.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
+install:
+ - mvn -s .tarvis-settings.xml clean install
```
Comment (asoldano) [inline .travis.yml]: Is this mvn clean install really needed? Why not simply running the mvn integration-test below in the 'script' phase ?

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+script:
+ - mvn -s .tarvis-settings.xml -B -fae -P$SERVER_VERSION integration-test
```
Comment (asoldano) [inline .travis.yml]: Can you fix the typo in the name of the setting file? :-) (s/tarvis/travis/)

Comment (asoldano) [reply .travis.yml]: @rsearls the name also needs fix in the command here above

File: .travis.yml [hunk 4]
```diff
@@ -9,8 +9,8 @@ jdk:
   - oraclejdk8
   - oraclejdk9
 env:
-  - SERVER_VERSION=wildfly1010
-  - SERVER_VERSION=wildfly1000
+ ## - SERVER_VERSION=wildfly1010
+ ## - SERVER_VERSION=wildfly1000
```
Comment (asoldano) [inline .travis.yml]: why are you completely commenting out the 1010 and 1000 runs? This way only 1100 runs are executed

**Comment by rsearls** [comment]
Ignore all file changes until I notify you.   In order for me to test
changes to these files
I have to submit them a my origin/master.   Not sure why you are seeing
this changes.

**Comment by rsearls** [comment]
The requested changes have been made and squashed back into PR 38.

**Comment by asoldano** [comment]
@rsearls , merged, thanks!

**Pull request merged**